### PR TITLE
[release] docs: update documentation with bundle

### PR DESCRIPTION
**Pull request by immortal-tofu**

### CODE REVIEW

File: docs/guides/frontend/node.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ const { createInstance } = require("fhevmjs");
 const createFhevmInstance = async () => {
   return createInstance({
     chainId: 11155111, // Sepolia chain ID
-    networkUrl: "https://rpc.sepolia.org/", // Sepolia RPC URL
-    gatewayUrl: "https://gateway.zama.ai",
+    networkUrl: "https://eth-sepolia.public.blastapi.io", // Sepolia RPC URL
```
Comment (poppyseedDev) [inline docs/guides/frontend/node.md:39]: is this an official url that is less likely to be deprecated ?

File: docs/getting_started/write_contract/remix.md [hunk 2]
```diff
@@ -16,9 +16,7 @@ This document provides guidance on using the new Zama plugin for the [the offici
 After connecting to the Zama Plugin, follow the steps to configure it:
```
Comment (yuxizama) [inline docs/getting_started/write_contract/remix.md]:
```suggestion
After connecting to the Zama pugin, follow the steps to configure it:
```

File: docs/getting_started/write_contract/remix.md [hunk 3]
```diff
@@ -16,9 +16,7 @@ This document provides guidance on using the new Zama plugin for the [the offici
...
 
 1. Click on the plugin button located on the left of the screen
-2. Add a Gateway URL to be able to request reencryption of ciphertexts, as shown in the picture below.
-
-The default recommended Gateway URL is: `https://gateway.devnet.zama.ai`.
+2. Use the Zama Sepolia settings or use a custom gateway and contracts settings.
```
Comment (yuxizama) [inline docs/getting_started/write_contract/remix.md]:
```suggestion
2. Use the **Zama Coprocessor - Sepolia** settings or use a custom gateway and contracts settings.
```

Comment (poppyseedDev) [inline docs/getting_started/write_contract/remix.md]: How do they know which contracts settings to use?

**Review by poppyseedDev — APPROVED** [review]

**Review by yuxizama — COMMENTED** [review]
Suggested minor adjustments, thanks!

File: docs/guides/frontend/node.md (hunk 1 above)
Comment (immortal-tofu) [reply docs/guides/frontend/node.md:39]: AFAIK, "https://rpc.sepolia.org/" is not a real RPC endpoint, no? I think giving a working one can avoid support. People simply copy/paste.

File: docs/getting_started/write_contract/remix.md (hunk 3 above)
Comment (immortal-tofu) [reply docs/getting_started/write_contract/remix.md]: It's preconfigured for Sepolia Zama. `Custom` is just a selection if they use a different coprocessor (today, there are none)

File: docs/getting_started/write_contract/remix.md (hunk 2 above)
Comment (immortal-tofu) [reply docs/getting_started/write_contract/remix.md]: Invalid correction, I'll fix it

**Pull request merged**